test/common/CMakeLists.txt

```diff
@@ -35,6 +35,10 @@ PCL_ADD_TEST(common_copy_make_borders test_copy_make_borders FILES test_copy_mak
 PCL_ADD_TEST(common_bearing_angle_image test_bearing_angle_image FILES test_bearing_angle_image.cpp LINK_WITH pcl_gtest pcl_common)
 PCL_ADD_TEST(common_polygon_mesh test_polygon_mesh_concatenate FILES test_polygon_mesh.cpp LINK_WITH pcl_gtest pcl_common)
 PCL_ADD_TEST(common_point_type_conversion test_common_point_type_conversion FILES test_point_type_conversion.cpp LINK_WITH pcl_gtest pcl_common)
+PCL_ADD_TEST("common_point_type_construction_runtime"  "test_common_point_type_construction_runtime" FILES test_point_type_construction.cpp LINK_WITH pcl_gtest pcl_common)
+target_compile_definitions("test_common_point_type_construction_runtime" PRIVATE PCL_RUN_TESTS_AT_COMPILE_TIME=false)
+PCL_ADD_TEST("common_point_type_construction_compiletime"  "test_common_point_type_construction_compiletime" FILES test_point_type_construction.cpp LINK_WITH pcl_gtest pcl_common)
+target_compile_definitions("test_common_point_type_construction_compiletime" PRIVATE PCL_RUN_TESTS_AT_COMPILE_TIME=true)
```

Comment:
My purpose to add a custom target was to keep shenanigans like this hidden.

Reply:
ok, i added that macro, though i changed its name to PCL_ADD_COMPILETIME_AND_RUNTIME_TEST, not to be misleading about the fact what tests would it actually run (PCL_ADD_STATIC_TEST should mean only static_assert-based tests) 
also given now there's a macro for combination of two types of tests, PCL_ADD_TEST should rather be called PCL_ADD_RUNTIME_TEST, but making some many changes is probably not worth it